libpod/pod_api.go

```diff
@@ -426,13 +426,18 @@ func (p *Pod) Inspect() (*PodInspect, error) {
 		}
 		podContainers = append(podContainers, pc)
 	}
+	infraContainerID := p.state.InfraContainerID
+	if err != nil {
```

Comment:
This can't return an error, though? Leftover error check?

Reply:
can't be too careful :) (whoopsies haha)